src/data/software-ides.ts

```diff
@@ -31,6 +31,73 @@ const softwareIDEs: IconData[] = [
         height: 36,
         xOffset: 0,
     },
+
+    {
+        name: ['vscode', 'Visual Studio Code'],
```

Comment:
Small nitpick; change capitalisation to "VSCode" as the first string is the exact way it would show up with `includeNames=true`. Also, spaces in the name will not work as they cannot be typed in the URL anyways.